lisa/sut_orchestrator/azure/transformers.py

```diff
@@ -573,7 +573,11 @@ def _internal_run(self) -> Dict[str, Any]:
             runbook.gallery_image_location,
         )
 
-        sig_url = f"{gallery.name}/{runbook.gallery_image_name}/{gallery_image_version}"
+        sig_url = (
+            f"{runbook.gallery_name}/"
```

Comment:
is it mandatory? if no, it may be an empty value.

Reply:
If there is no `gallery_name `configured in the runbook, `SigTransformerSchema `has a default value "`lisa_default_gallery`". So it won't be empty.